src/components/ContentSidebar/ActivityFeed/task-form/TaskForm.js

```diff
@@ -0,0 +1,312 @@
+/**
```

Comment:
this file is based on ApprovalCommentForm

Reply:
How similar is this file to ApprovalCommentForm? Can we avoid duplicating it?

Reply:
It's increasingly divergent and would require a lot of layout props like "form element order", "enable mentions", etc.

Reply:
Can we extract any additional common sub-components or behaviors to reduce the duplication?

Reply:
The data layer would make the most sense to refactor, the forms are pretty similar. Will see at the end if there's anything to extract. Also would be better to refactor after integration tests are written.